kolla/defaults.yml

```diff
@@ -31,14 +31,15 @@ switch_configs: []
 neutron_networks: []
 
 enable_mariadb: yes
+enable_mariabackup: yes
 enable_chrony: yes
 enable_fluentd: yes
 enable_memcached: yes
 memcached_max_memory: 2048
 enable_rabbitmq: yes
 
 # Automatic backups
-backup_dir: /var/db/backup
+backup_dir: /var/lib/docker/volumes/mariadb_backup/_data
```

Comment:
Why is the host path to the volume being used here? This path depends on the docker volume driver that's in use, and generally shouldn't be used directly, instead favoring a docker mount of the volume.